lib/cosmos/tools/cmd_sender/cmd_sender.rb

```diff
@@ -145,7 +156,12 @@ def initialize_actions
       connect(@show_ignored, SIGNAL('toggled(bool)'), self, SLOT('update_cmd_params(bool)'))
 
       @cmd_raw = Qt::Action.new(tr('Disable &Parameter Conversions'), self)
-      @cmd_raw.statusTip = tr('Send the command without running write or state conversions')
+      tip = 'Send the command without running write or state conversions'
+      if @production
+        tip += ' - Disabled in Production Mode'
+        @cmd_raw.setEnabled(false)
+      end
+      @cmd_raw.statusTip = tr(tip)
```

Comment:
Just a comment that you don't need to use tr().  We don't use it consistently and will probably never support other languages.